RELEASES

```diff
@@ -5,6 +5,44 @@ A new release happens every 6 weeks:
 - 1st day on the first month of each quarter
 - 15th day of the second month of each quarter
 
+
+## v4.0.0 (2018-01-26)
```

Comment:
2019?

Reply:
Thank you, I mess something up every release.